tools/c7n_azure/tests/cassettes/ArmResourceTest.test_metric_filter_not_find_average.yaml

```diff
@@ -1,123 +1,493 @@
-        0gf0+zr/6NH3fvFHxYw+vNusJ820LlZtUS2bu1n28ODhg/3J9v3ZXra9P3t4sJ09OMi27x3snd97
-        sHt/tvMgv1vnTbWup/nndbVeNXffnL5+8/v/5Bd3V3V1Wczyurn7RTGtq6Y6b8cn1WK1bvO7l0Xd
-        rrPyi2w6L5Z5c3c6bfOmvVxEXzpbNsXFvG3uLvK2LqbN3Zc59bZss4s8PXn5FQ2ivV7RGD4afoXa
-        LLMFtfnFZsAf9YCU1TQrix/ks5+Mt/glo4/Wy4Kob76gd3gW8rrIGyYh9ZbNsjbjPvDR90cf4W/6
-        7Rdz29dttlgRBDM7dsp2djA72WVeU3cfPdobP7j/6f17e59+eu/e7qf3D+49/CXfx38yDJr3KTXy
-        xjtAVwJpJqfOL2hG6SX6s50D/Tor181Hv+T/AelhbU1UAgAA
-    headers:
-      Cache-Control: [no-cache]
-      Content-Encoding: [gzip]
-      Content-Type: [application/json]
-      Date: ['Mon, 10 Sep 2018 19:52:54 GMT']
-      Server: [Microsoft-IIS/8.5]
-      Strict-Transport-Security: [max-age=31536000; includeSubDomains]
-      Vary: ['Accept-Encoding,Accept-Encoding']
-      X-AspNet-Version: [4.0.30319]
-      X-Content-Type-Options: [nosniff]
-      X-Powered-By: [ASP.NET]
-      __HandlingServerId__: [shoeboxproxyustwo_shoeboxproxyustwo-red_MetricsMP_IN_6]
-      x-ms-correlation-request-id: [d6a19b10-cafb-4561-9390-75befa7a24cf]
-      x-ms-ratelimit-remaining-subscription-reads: ['11993']
-      x-ms-request-id: ['{d6a19b10-cafb-4561-9390-75befa7a24cf}']
-      x-ms-routing-request-id: ['WESTUS2:20180910T195255Z:d6a19b10-cafb-4561-9390-75befa7a24cf']
-    status: {code: 200, message: OK}
-version: 1
+{
+    "version": 1,
+    "interactions": [
+        {
+            "request": {
+                "method": "GET",
+                "uri": "https://management.azure.com/subscriptions/ea42f556-5106-4743-99b0-c129bfa71a47/providers/Microsoft.Compute/virtualMachines?api-version=2019-03-01",
+                "body": null,
+                "headers": {
+                    "Accept-Encoding": [
+                        "gzip, deflate"
+                    ],
+                    "Accept": [
+                        "application/json"
+                    ]
+                }
+            },
+            "response": {
+                "status": {
+                    "code": 200,
+                    "message": "OK"
+                },
+                "headers": {
+                    "pragma": [
+                        "no-cache"
+                    ],
+                    "vary": [
+                        "Accept-Encoding"
+                    ],
+                    "date": [
+                        "Fri, 26 Apr 2019 18:07:32 GMT"
+                    ],
+                    "content-type": [
+                        "application/json; charset=utf-8"
+                    ],
+                    "expires": [
+                        "-1"
+                    ],
+                    "x-ms-ratelimit-remaining-resource": [
+                        "Microsoft.Compute/HighCostGet3Min;133,Microsoft.Compute/HighCostGet30Min;590"
+                    ],
+                    "cache-control": [
+                        "no-cache"
+                    ],
+                    "transfer-encoding": [

```

Comment:
we shouldn't be recording this since its not chunked after the serialization, this should probably go into the omit headers.